test/bin/ci_phase_iso_build.sh

```diff
@@ -116,7 +116,6 @@ cd "${ROOTDIR}"
 # Get firewalld and repos in place. Use scripts to get the right repos
 # for each branch.
 $(dry_run) bash -x ./scripts/devenv-builder/configure-vm.sh --no-build --force-firewall "${PULL_SECRET}"
-$(dry_run) bash -x ./scripts/image-builder/configure.sh
```

Comment:
Problem with this is that `build_rpms.sh` uses things installed in this script, like `parallel`. We could move whole `if ${COMPOSER_CLI_BUILDS}` to this place,  but we also need to install some things for bootc, so starting osbuild workers is a bit pointless, but if we want to keep the script `manage_composer_config.sh create` then we need to do it, I guess

Reply:
I think an alternative would be to have separate args:
```
manage_composer_config.sh create
manage_composer_config.sh start-osbuild-workers N
```
So we always execute first one, and we only execute second one `if ${COMPOSER_CLI_BUILDS}`

Reply:
I added a command `manage_composer_config.sh create-workers <num>` to just create the osbuild workers.

Reply:
It's a good approach. Small nit: let's get rid of the `<num> arg as I commented below?